__init__.py

```diff
@@ -29,21 +29,23 @@
 
 from neon_utils.skills.neon_skill import NeonSkill, LOG
 from mycroft.skills.core import intent_file_handler
-from .request_handling import RequestHandler
-from .request_handling import existing_lang_check, get_shop_data,\
-                                shop_selection_by_floors,\
-                                location_format,\
-                                curent_time_extraction
+from .request_handling import existing_lang_check, get_store_data, \
+    store_selection_by_floors, \
+    location_format, \
+    curent_time_extraction
 import re
+import lingua_franca
+from lingua_franca.format import nice_duration
 
+lingua_franca.load_language('en')
 
 
 class DirectorySkill(NeonSkill):
 
     def __init__(self):
         super(DirectorySkill, self).__init__(name="DirectorySkill")
         self.url = "https://www.alamoanacenter.com/en/directory/"
-
+        self.request_lang = self.lang.split('-')[0]
```

Comment:
Language is handled per-Message and globally only as a fallback. `self.lang` will resolve the correct language, but `self.request_lang` here will remain set to the default value; you could convert this to a property if this change was helpful

```python
@property
def request_lang(self):
    return self.lang.split('-')[0]
```

The difference here is that `self.lang` will resolve the `Message` associated with a particular response any time its referenced and use that to get the correct language